src/test/integration/redis/tests/test_redis.py

```diff
@@ -0,0 +1,45 @@
+import os
+import sys
+import time
+import unittest
+from testcontainers.redis import RedisContainer
+import subprocess
+
+from test.test_utils.spans_parser import SpansContainer
+
+
+class TestRedisSpans(unittest.TestCase):
+    def test_redis_instrumentation(self):
+        with RedisContainer("redis:latest") as redis_server:
+            example_path = os.path.join(
+                os.path.dirname(os.path.abspath(__file__)), "../app/redis_example.py"
+            )
+            subprocess.check_output(
+                [sys.executable, example_path],
+                env={
+                    **os.environ,
+                    "REDIS_HOST": redis_server.get_container_host_ip(),
+                    "REDIS_PORT": str(redis_server.get_exposed_port(6379)),
+                    "AUTOWRAPT_BOOTSTRAP": "lumigo_opentelemetry",
+                    "OTEL_SERVICE_NAME": "app",
+                },
+            )
+            # TODO Do something deterministic
```

Comment:
out of scope, not blocking: your comment on this in the js distro was on point, we should have a shared utility method to replace all of these.... if you have time/energy, cool, otherwise one of us should do that next time we touch this repo

Reply:
abs right!
I want to push it forward, but will hopefully do it in my next task here!